app/Http/Requests/TaskComment/DestroyTaskCommentRequest.php

```diff
@@ -16,6 +16,8 @@ public function _authorize(): bool
 
     public function _rules(): array
     {
-        return [];
```

Comment:
without this rule, the ItemController does not find the id key

Reply:
Because you have changed it at https://github.com/cattr-app/backend-application/pull/45/files/23ec22f04a37fe857d4484bcb9fc6441a524876a#diff-fc8205c776dae5f3ecd772e0b66cfdebbb0c50780abab479bd4a47d8bd711044R161

Reply:
![image](https://user-images.githubusercontent.com/51990904/227733118-da942da4-85c0-4a23-8f0f-e32ba4163625.png)
![image](https://user-images.githubusercontent.com/51990904/227733139-cd7031a9-6604-4381-aab9-aed701ebdb35.png)
![image](https://user-images.githubusercontent.com/51990904/227733152-f9cd8f3b-354e-44ce-a299-34edf78b6356.png)
for example DestroyTaskRequest.php too have id param in rules
![image](https://user-images.githubusercontent.com/51990904/227733248-e775b6ce-6d27-4679-ba4a-3af34e7f3a96.png)

Reply:
without id param in rules logic for destroy do not work